checks/class-skip-links.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Check if skip links are present on block themes
+ *
+ * @package Theme Check
+ */
+
+/**
+ * Check if skip links are present on block themes
+ */
+class Skip_Links_Check implements themecheck {
+	/**
+	 * Error messages, warnings and info notices.
+	 *
+	 * @var array $error
+	 */
+	protected $error = array();
+
+	/**
+	 * Returns true if the theme is a block theme.
+	 *
+	 * @var array $is_block_theme
+	 */
+	protected $is_block_theme = false;
+
+	/**
+	 * The WP_Theme instance being checked.
+	 *
+	 * @var WP_Theme $wp_theme
+	 */
+	protected $wp_theme = false;
+
+	function set_context( $data ) {
+		if ( isset( $data['theme'] ) ) {
+			$this->wp_theme       = $data['theme'];
+			$this->is_block_theme = wp_is_block_theme();
+		}
```

Comment:
As far as I can see `is_block_theme` is not being used in this check class. Is it used somewhere?